simpleflow/swf/process/worker/base.py

```diff
@@ -173,7 +182,7 @@ def dispatch(self, task):
         name = task.activity_type.name
         return self._dispatcher.dispatch_activity(name)
 
-    def process(self, poller, token, task):
+    def process(self, poller, token, task, middlewares=None):
```

Comment:
```suggestion
    def process(self, poller, token, task, middleware=None):
```
or
```suggestion
    def process(self, poller, token, task, middleware_module=None):
```
or allow multiple middlewares :wink: